lib/sprangular.rb

```diff
@@ -6,21 +6,21 @@
 require 'slim'
 require 'rabl'
 require 'jquery-rails'
+require 'rails-assets-underscore'
+require 'rails-assets-underscore.string'
 require 'ngannotate/rails'
 require 'prerender_rails'
 require 'rails-assets-angular'
 require 'rails-assets-angular-route'
 require 'rails-assets-angular-sanitize'
 require 'rails-assets-angular-animate'
-require 'rails-assets-bootstrap-sass-official'
 require 'rails-assets-underscore'
 require 'rails-assets-underscore.string'
-require 'rails-assets-angular-strap'
-require 'rails-assets-angular-motion'
 require 'rails-assets-angular-translate'
 require 'rails-assets-angular-i18n'
-require 'rails-assets-bootstrap-additions'
 require 'rails-assets-angularytics'
+require 'bourbon'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.